### ISSUE TITLE
Flag URLs known to be blocked by court order

### ISSUE THREAD

**Issue by gwire** [openrightsgroup/cmp-issues#67]
Some URLs are known to be blocked by court order on some networks e.g. thepiratebay.se

We should maintain a list of these (if we don't already)

a) Then the results page
https://www.blocked.org.uk/results?url=http%3A%2F%2Fthepiratebay.se
should include a banner that mentions this fact - possibly linking to an entry on ORG or 451unavailable wiki.

b) See networks that are proactively blocking these sites without being subject to court orders.

c) If sites are blocked because of court orders, but we don't have them in the list, make it possible for people to flag that URL as related to a court order.  This way we can attempt to reverse-crowd source the contents of the blocking orders (which aren't public).

**Comment by graphiclunarkid** [comment]
I'm closing #46 in favour of this issue, as @gwire has provided a better description of the feature here. Not sure if @dantheta's proposed solution in #46 is still relevant however we can refer to that ticket if it is.

**Comment by gwire** [comment]
We are currently maintaining information about current UK blocking orders at http://wiki.451unavailable.org.uk/ - so discovering any new information (alternate domain names etc) would be useful.

Should be quite easy to detect - for example Virgin Media uses http://assets.virginmedia.com/site-blocked.html rather than http://assets.virginmedia.com/parental.html as its blocking page.

**Comment by Deku-shrub** [comment]
Right, so this ties in quite neatly with my unfinished project at http://metaproxy.net/

I scrape list of Pirate Bay proxies from the following sources which I test then put into my 'metaproxy':
http://proxybay.info/list.txt
http://www.piratebayproxylist.com/2012/05/pbpl-pirate-bay-proxy-list.html
http://ukbay.org/
https://encyclopediadramatica.se/The_Pirate_Bay/thepiratebay.se_proxy_list <- beware it's a wiki

Serving 302's in not a crime AFAIK :)

Since I've decided (subject to not being banned from doing so by ORG) to use blocked.org.uk as my probe (easier then testing from a single home connection) I should effectively fill the blocked database with pirate bay proxy data.

**Comment by graphiclunarkid** [comment]
@Deku-shrub Feel free to use Blocked for this. If it would be easier to have direct API access then you need to speak to @dantheta. We should continue to discuss how best to organise and present the data on blocked.org.uk though.
